### ISSUE TITLE
Front page redesign

### ISSUE THREAD

**Issue by KacperM45440** [fishfolk/fishfolk.org#8]
**The panel**

As of right now, social media buttons are displayed on an opaque panel with plain text. To keep them aligned with the "simple but smart" style, we could instead completely remove the panel and replace it with bigger icons of the respective platforms (example below - buttons don't necessarily have to be yellow).

![image](https://user-images.githubusercontent.com/104095161/223858223-2b6bb38a-c47e-4c01-ae11-7a83e0a9eb90.png)

**The font**

To increase visual clarity and variety of text and site elements, a thing to look into would be to introduce other fonts alongside the current pixelated one. Nothing specific in mind, but we'd definitely need something looks good both on its own, bordered and bolded.

**The fish**

Randomly selected fish are an awesome addition to the page, but they could use some extra touches so that it actually looks like they're holding their weapons. I think they could also be moved *just a couple* of pixels away from the center, so that their fins aren't covered by the text.

**Comment by zicklag** [comment]
> Randomly selected fish are an awesome addition to the page, but they could use some extra touches so that it actually looks like they're holding their weapons.

Good idea. We can use the relatively new fish/fin sprite separation that we use in the game now to make the fish hold the items.

> I think they could also be moved just a couple of pixels away from the center, so that their fins aren't covered by the text.

👍

> As of right now, social media buttons are displayed on an opaque panel with plain text. To keep them aligned with the "simple but smart" style, we could instead completely remove the panel and replace it with bigger icons of the respective platforms (example below - buttons don't necessarily have to be yellow).

Yeah, I like the idea of replacing them with the icons. And we should still be able to play with the mouseover effect to add just a little bit of juice to them, too. Probably not the hue shift, but a bounce effect would be great I think.

> To increase visual clarity and variety of text and site elements, a thing to look into would be to introduce other fonts alongside the current pixelated one. Nothing specific in mind, but we'd definitely need something looks good both on its own, bordered and bolded.

Yeah, the font is super easy to swap, we just need to pick one or more out for different fonts for different parts of the site.

**Comment by zicklag** [comment]
I opened a few issues, to cover these different aspects: #9, #10, #11.

I think I can do #11 and #9 after I get https://github.com/fishfolk/jumpy/issues/494 done.

@KacperM45440 maybe you can help picking out a font, and choosing where exactly we might want to put the new font for #10? Then I can add it to the site, or you can if you know how and want to do it.
